src/config_file.c

```diff
@@ -631,7 +631,9 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 		if (c->data_directory) {
 			c->statusfile_path = g_build_filename(c->data_directory, "central.raucs", NULL);
 		} else {
-			g_message("Config option 'statusfile=<path>/per-slot' unset, falling back to per-slot status");
+			g_message("No data directory or status file set, falling back to per-slot status.\n"
+					"This will limit supported features.\n"
+					"Consider setting 'data-directory=<path>' or 'statusfile=<path>/per-slot' explicitly.");
```

Comment:
`per-slot` will still limit supported features. Perhaps the limited features message should be triggered by `per-slot` only?

Reply:
So we should probably just omit the limited feature line. For `per-slot` case in general I already I introduced the follow-up message that says

> Using per-slot statusfile. System status information not supported!

I'll also adapt the commit message accordingly